### PR TITLE
Support for role array in AWS managed policies

### PR DESCRIPTION
**Pull request by raphaellondner-okta**

### CODE REVIEW

File: src/main/java/com/okta/tools/awscli.java [hunk 1]
```diff
@@ -553,17 +553,52 @@ private static String ProcessPolicyDocument(String policyDoc) {
...
+        }
+
+        //Prompt user for policy selection
+        int selection = numSelection(lstRoles.size());
+
+        if(selection < 0 && lstRoles.size() > selection) {
+            System.out.println("\nYou entered an invalid number. Please try again.");
+            return SelectRole(lstRoles);
+        }
+        else {
```
Comment (ibabiy-okta) [inline src/main/java/com/okta/tools/awscli.java]: nit: you don't need `else` clause here, because you have return in `if`

**Comment by ibabiy-okta** [comment]
@raphaellondner-okta, since you requested to take a look only at the AWS logic part, I won't comment other stuff. 
The overall AWS logic is good, but I suggest to run a Okta's PMD/Codestyle checks (like in okta-core). It will help you to identify all those things.

👍

**Comment by raphaellondner-okta** [comment]
Thanks @ibabiy-okta for the feedback. I have removed the unnecessary else statement in the [latest commit](https://github.com/oktadeveloper/okta-aws-cli-assume-role/pull/20/commits/28eb925e58bd8f2c11ecd9532d9868ae9b944841).

**Comment by ibabiy-okta** [comment]
🚢

**Pull request merged**